x-pack/plugins/infra/types/eui.d.ts

```diff
@@ -201,4 +201,17 @@ declare module '@elastic/eui' {
     idSelected?: any;
   };
   export const EuiButtonGroup: React.SFC<EuiButtonGroupProps>;
+
+  type EuiSuperSelectProps = CommonProps & {
```

Comment:
That's an interesting change. You have it in `plugins/infra/types/eui.d.ts`.  I assume you want it here for the super select though and that later it will be merged from master into our branch.